src/domain/entities/DhisDataPackage.ts

```diff
@@ -1,3 +1,5 @@
+import { D2Geometry } from "@eyeseetea/d2-api/schemas";
```

Comment:
We should not have any reference to the data layer in our domain. Let's create a Geometry entity and use it instead the one from d2-api.